packages/relay/tests/lib/hapiService.spec.ts

```diff
@@ -103,9 +103,10 @@ describe('HAPI Service', async function () {
     });
   });
 
-  withOverriddenEnvsInMochaTest({ HAPI_CLIENT_ERROR_RESET: '[50]' }, () => {
+  withOverriddenEnvsInMochaTest({ HAPI_CLIENT_ERROR_RESET: [50] }, () => {
     it('should be able to reinitialise SDK instance upon error status code encounter', async function () {
-      const hapiClientErrorReset: Array<number> = JSON.parse(ConfigService.get('HAPI_CLIENT_ERROR_RESET'));
+      // Use type assertion for backward compatibility until all code is updated
```

Comment:
q: where is the type assertion?

Reply:
I planned to separate the new types and replacing them in the code in separate PR's initially so this is a leftover comment. I will remove it.